xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
> There is 0.1 sek between the change: Is this OR or is this AND?

AND because there are up to 8 secs in the message queue. Maybe you should just add some mode that VideoPlayer prevents from doing a resync for this case.
I have not seen any content so far that would require this. What is it?